tools/hAMRonize2AntiSMASH/hAMRonize2AntiSMASH.xml

```diff
@@ -0,0 +1,21 @@
+<tool id="hAMRonize2AntiSMASH" name="Convert hAMRonize to AntiSMASH" version="1.0.0">
+  <requirements><requirement type="package" version="3.9">python</requirement></requirements>
+  <command detect_errors="exit_code"><![CDATA[
+  ln -s '$input' input.json;
```

Comment:
please use && instead of ;

Reply:
@BartvdHout-TimKuijten @TimKAvans please add `profile="21.09"` on line 1. This will also solve this issue.